### PR TITLE
sub: add justify of subtitles

### PR DESCRIPTION
**Pull request by DanOscarsson**
To make it easier for the eyes, multi line subtitles should
be left justified (for most languages).
This adds an option to define how subtitles are to be justified
indendently of how they are aligned.
Also add option to enable --sub-justify to be applied on ASS subtitles.

I agree that my changes can be relicensed to LGPL 2.1 or later.

### CODE REVIEW

File: sub/sd_ass.c [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 #include <libavutil/common.h>
 #include <ass/ass.h>
+#include <ass/ass_types.h>
```
Comment (wm4) [inline sub/sd_ass.c]: Stray change? Or is it needed?

Comment (DanOscarsson) [inline sub/sd_ass.c]: To be able to test on ASS_JUSTIFY_AUTO.
But I see that libass now has released a new version so I could change to test on LIBASS_VERSION instead and then this include is not needed.

Comment (wm4) [inline sub/sd_ass.c]: I just checked, and ass.h indeed already includes ass_types.h. Remove it and I can merge this.

Comment (DanOscarsson) [inline sub/sd_ass.c]: Yes, it does that now. Removed.

**Comment by wm4** [comment]
Merged.

**Pull request closed without merging**